include/mbgl/ios/MGLMapView.h

```diff
@@ -498,10 +498,10 @@ IB_DESIGNABLE
 
 /**
  Sets the visible region so that the map displays the specified annotations.
- 
+
  Calling this method updates the value in the visibleCoordinateBounds property
- and potentially other properties to reflect the new map region.
- 
+ and potentially other properties to reflect the new map region. Defaults to insetting the annotation bounds by 100,100,100,100, or if map is smaller than 200x200 in either dimmension, sets the insets to 20% of that dimmension on either side.
```

Comment:
"Dimension" is spelled incorrectly. We'll also want to wrap this at 80 characters.